app/src/docker/docker-compose.yml

```diff
@@ -182,3 +183,29 @@ services:
     networks:
       - internal
 
+  vro-console:
+    image: va/abd_vro-console:latest
+    profiles: ["debug"]
+    stdin_open: true
+    tty: true
+    environment:
+      # TODO: move these Postgres and RabbitMQ configurations into a reusable yaml block
+      POSTGRES_HOST: vro-api-postgres-service
+      POSTGRES_DB: ${POSTGRES_DB}
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_URL: ${POSTGRES_URL} #jdbc:postgresql://vro-api-postgres-service:5432/vro
+      POSTGRES_SCHEMA: claims
+      RABBITMQ_PLACEHOLDERS_HOST: vro-api-rabbit-mq-service
+      RABBITMQ_PLACEHOLDERS_USERNAME: ${RABBITMQ_PLACEHOLDERS_USERNAME}
+      RABBITMQ_PLACEHOLDERS_USERPASSWORD: ${RABBITMQ_PLACEHOLDERS_USERPASSWORD}
```

Comment:
Needs to use the reusable block.

Reply:
Yup, the reusable blocks PR wasn't merged yet. Updating now.

Reply:
@yoomlam Can you also correct names in vro-console depends-on block?